chart/values.yaml

```diff
@@ -30,6 +30,19 @@ env:
         name: hcloud
         key: token
 
+  ROBOT_USER_NAME:
```

Comment:
Since we use the `h` `cloud` convention to refer to Hetzner Cloud resources, what about using `h` `robot` to refer to any Hetzner Robot resources? 

e.g HCLOUD_TOKEN => HROBOT_USER_NAME

Or we should have CLOUD_TOKEN => ROBOT_USER_NAME.

Reply:
In my mind, `hcloud` is an established abbreviation for Hetzner Cloud, as used in the `cli` and this projects name and many other places.

I have never heard anyone call it `hrobot` (besides the Go SDK). Its always Robot or Hetzner Robot.

This is also a compatibility thing, these [env var names were used in the Syself fork](https://github.com/syself/hetzner-cloud-controller-manager/blob/04b701babcebdffda9cb869e24416cc9f8998fc3/hcloud/cloud.go#L53-L54), so I would prefer to keep them as they are.